experiments/integrated/vaira_ranch/vaira_domain.jl

```diff
@@ -0,0 +1,22 @@
+# Domain setup
+# For soil column
+nelements = 20
+zmin = FT(-10)
+zmax = FT(0)
+dz_bottom = FT(1.0)
+dz_top = FT(0.05)
+
+land_domain = Column(;
+    zlim = (zmin, zmax),
+    nelements = nelements,
+    dz_tuple = (dz_bottom, dz_top),
+)
+canopy_domain = ClimaLSM.Domains.obtain_surface_domain(land_domain)
+
+# Number of stem and leaf compartments. Leaf compartments are stacked on top of stem compartments
+n_stem = Int64(0)
+n_leaf = Int64(1)
+h_leaf = FT(0.8) # m
```

Comment:
Is this ok? I saw 0.7 m in this publication (https://gmd.copernicus.org/articles/8/295/2015/gmd-8-295-2015.pdf). Could this impact the collapse of H before expected?

Reply:
I can change it to 0.7.

Changes from the other simulation: h_leaf from 0.2 m -> 0.8m and h_atmos from 32 m(Ozark site flux tower height) to 2m (guess for this site). I think it was the change in h_atmos that really affected the fluxes, because this is a change of 30m. The displacement height is ~0.7*h_canopy, so the change in that due to change in h_canopy much smaller

Reply:
@braghiere is this ok to merge then?

Reply:
Ok with me. Thanks.